src/devices/machine/cs8900a.h

```diff
@@ -0,0 +1,114 @@
+// license:GPL-2.0+
+// copyright-holders:Rhett Aultman
```

Comment:
You need to credit the original author(s) or copyright assignee(s) here in addition to yourself in case someone wants to track down all copyright holders for a licensing issue.